runestone/.gitattributes

```diff
@@ -0,0 +1,17 @@
+# Auto detect text files and perform LF normalization
```

Comment:
I'm very leary to accept anything that messes with end of line stuff.  I've run into so many problems with this totally mesing up a repo over the years.